src/components/ContactPolicy/ContactPolicy.js

```diff
@@ -42,14 +43,17 @@ export const ContactPolicy = InjectAppServices(
       } else if (formSubmitted) {
         message.text = _('contact_policy.success_msg');
         message.type = 'success';
+        message.delay = 3000;
```

Comment:
It would be nice to have this delay set as a global variable for success delay somewhere, to avoid repeating this value each time we use a success message. Or have this delay included inside the `Flash` component. What do you think?

Reply:
Se pudiese crear eso de la variable global y que se utilice por defecto si no se le específica un delay custom ...

Reply:
I think it would be nice to have it set as a global variable.